src/VirtoCommerce.ContentModule.Web/Controllers/Api/ContentController.cs

```
@@ -247,6 +248,11 @@ public async Task<ActionResult> Unpack(string contentType, string storeId, [From
         [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
         public async Task<ActionResult> CreateContentFolder(string contentType, string storeId, [FromBody] ContentFolder folder)
         {
+            if (!new ContentFolderValidator().Validate(folder).IsValid)
```

Should we return an error in case of unsuccessful validation?